modules/apps/marketplace/marketplace-app-manager-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -32,13 +30,16 @@ PortalUtil.addPortletBreadcrumbEntry(request, LanguageUtil.get(request, "app-man
 />
 
 <clay:management-toolbar
+	clearResultsURL="<%= viewAppsManagerManagementToolbarDisplayContext.getClearResultsURL() %>"
 	filterDropdownItems="<%= viewAppsManagerManagementToolbarDisplayContext.getFilterDropdownItems() %>"
+	filterLabelItems="<%= viewAppsManagerManagementToolbarDisplayContext.getFilterLabelItems() %>"
+	itemsTotal="<%= viewAppsManagerManagementToolbarDisplayContext.getItemsTotal() %>"
 	searchActionURL="<%= viewAppsManagerManagementToolbarDisplayContext.getSearchActionURL() %>"
 	searchContainerId="appDisplays"
 	searchFormName="searchFm"
 	selectable="<%= false %>"
 	showSearch="<%= true %>"
-	sortingOrder="<%= searchContainer.getOrderByType() %>"
+	sortingOrder="<%= viewAppsManagerManagementToolbarDisplayContext.getOrderByType() %>"
```

Comment:
The `managementToolbarDisplayContext` should have a `getSortingOrder` method on it.  I remember now, it basically returns the result of the method you made public above at https://github.com/drewbrokke/liferay-portal/pull/738/commits/e575dd0996849fa0dde0698b26207cba1658d28b#diff-f140e89a7ab0ed381fa150ca6a6ce60fR101.

See https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/display/context/BaseManagementToolbarDisplayContext.java#L103-L106.